### ISSUE TITLE
How to define the a custom output dir / filename?

### ISSUE THREAD

**Issue by paule96** [deanwiseman/DartSassBuilder#16]
Hi,

is it possible to define a custom output path?

**Comment by deanwiseman** [comment]
Hi, thanks for reaching out. Unfortunately this isn't supported at the moment

Could you please describe a basic use case for this type of behaviour? I'd like to understand the context a bit better

**Comment by paule96** [comment]
yes I have an example. I have a project where I have [here](https://github.com/codez-one/CZ.Azure.FileExchange/tree/main/src/CZ.Azure.FileExchange/Client/styles) the SCSS files. But the folder that should contain the `main.css` is [this](https://github.com/codez-one/CZ.Azure.FileExchange/tree/main/src/CZ.Azure.FileExchange/wwwroot).

You can see my current workaround [here](https://github.com/codez-one/CZ.Azure.FileExchange/blob/a73abe550e973e8d6ea0b817a3027686d9f8c93a/src/CZ.Azure.FileExchange/CZ.Azure.FileExchange.csproj#L31).

The problem with this workaround is, this is only working while build time, because I don't know how to find out if the is a refresh event from your task.

**Comment by deanwiseman** [comment]
Hi @paule96,

Thank you for providing some extra information and a project. I'll be taking a look at this and will provide any updates here in this issue.

**Comment by elibretto** [comment]
Hi @deanwiseman 

Did you manage to get anywhere with this? I have a similar use case to @paule96

**Comment by srad** [comment]
Yep, I have the same issue to define the exact output folder for the build. That is a very basic option for build steps to seperate source and dist files. So there is currently no solution as far as I can see. So I'm back to using the npm version I guess.
